### PR TITLE
Updated Windows installation instructions

### PR DESCRIPTION
**Pull request by Magnificus36**
Added link to 64bit Python version and added remark that pyreadline and Python should both be 64bits or 32bits.
(Also now link to Python 2.7.13 instead of 2.7.12)

### CODE REVIEW

**Comment by gpotter2** [comment]
@Magnificus36 Please rebase against upstream/master

File: doc/scapy/installation.rst [hunk 1]
```diff
@@ -323,7 +323,6 @@ ImageMagick (takes long to compile)
  # tar xvfz /tmp/ports.tar.gz 
  # cd /usr/ports/graphics/ImageMagick/
  # make install
-
```
Comment (gpotter2) [inline doc/scapy/installation.rst]: "Please avoid unnecessary changes"

Comment (Magnificus36) [inline doc/scapy/installation.rst]: Sorry, is now fixed.
(Went wrong while resolving merge conflict while using the GitHub webinterface.)

**Comment by guedou** [comment]
@gpotter2 are you fine with this PR ?

**Comment by gpotter2** [comment]
Looks fine
